main.jsx

```diff
@@ -373,24 +419,11 @@ function Links(props) {
 
 // NOTE to all: keep this list sorted alphabetically by name.
 const PARTNER_LOGOS = [
```

Comment:
For some of those groups (at the very least Alibaba and Vercel), the individuals who had contributed to WinterCG are no longer working there, and there's no one currently involved who's part of them.

Reply:
Sure, but we should be in contact with the organization before removing their logo. This may motivate them to continue, rather than feeling like they've been kicked out.